client/packages/system/src/Encounter/ListView/AdditionalInformationColumn.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { Typography } from '@common/components';
+import {
+  ColumnDefinition,
+  ColumnDescription,
+  EncounterNodeStatus,
+  RecordWithId,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useLogicalStatus } from '../utils';
+import {
+  EncounterRowFragment,
+  ProgramEnrolmentRowFragmentWithId,
+} from 'packages/programs/src';
+
+export const getEncounterEventCell = (rowData: EncounterRowFragment) => {
+  const t = useTranslation();
+  let additionalInfo = [];
+
+  if (!!rowData?.events[0]) {
+    additionalInfo.push(rowData.events[0].data ?? '');
+  }
+
+  if (rowData?.status === EncounterNodeStatus.Pending) {
+    const startDatetime = new Date(rowData?.startDatetime);
+    additionalInfo.push(useLogicalStatus(startDatetime, t));
+  }
+
+  return additionalInfo;
+};
+
+export const getProgramEventCell = (
```

Comment:
I would move this one to the program file.

Also, these could be renamed to something like `programAdditionalInfoAccessor`

Reply:
Then shouldn't the Encounter get moved to the Encounter file too? I just put them both here to keep everything in one place...

Reply:
[500ad64](https://github.com/openmsupply/open-msupply/pull/1940/commits/500ad64af89a16ac4d991c747b5b1ecf586045fb)